infrastructure/proto/log_replication_metadata.proto

```diff
@@ -57,60 +57,18 @@ message ReplicationMetadata {
 /*--------END---------*/
 
 /*-------- Replication Status Table V1------*/
-
 enum SyncType {
   SNAPSHOT = 0;
   LOG_ENTRY = 1;
 }
 
-/*
- * Replication Status Key
- */
-message ReplicationStatusKey {
-  string clusterId = 1;
-}
-
-/*
- * Replication Status Value
- * Source Site sets the completionPercent, Sink sets the dataConsistent boolean
- */
-message ReplicationStatusVal {
-  option (org.corfudb.runtime.table_schema).stream_tag = "lr_status";
-
-  uint64 remainingEntriesToSend = 1; // Set only by Source
-  bool dataConsistent = 2; // Set only by Sink
-  SyncType syncType = 3;
-  SyncStatus status = 4;
-  SnapshotSyncInfo snapshotSyncInfo = 5;
-}
-
-/*-------- Replication Status Table V2------*/
-
-message ReplicationStatus {
-  option (org.corfudb.runtime.table_schema).stream_tag = "lr_status";
-  oneof status {
-    SourceReplicationStatus source_status = 1;
-    SinkReplicationStatus sink_status = 2;
-  }
-}
-
-message SourceReplicationStatus {
-  option (org.corfudb.runtime.table_schema).stream_tag = "lr_status";
-
-  uint64 remainingEntriesToSend = 1;
-  ReplicationInfo replicationInfo = 2;
-}
-
-message SinkReplicationStatus {
-  option (org.corfudb.runtime.table_schema).stream_tag = "lr_status";
-  bool dataConsistent = 1;
-  ReplicationInfo replicationInfo = 2;
-}
-
-message ReplicationInfo {
-  SyncType syncType = 1;
-  SyncStatus status = 2;
-  SnapshotSyncInfo snapshotSyncInfo = 3;
+enum SyncStatus {
+  ONGOING = 0;
+  COMPLETED = 1;
+  ERROR = 2;
+  NOT_STARTED = 3;
+  STOPPED = 4;
+  UNAVAILABLE = 5;
```

Comment:
nit: I thought we can leave a comment here explaining something like V1 schemas are also moved to runtime/, keeping the old schema here for some rolling upgrade consideration?

Reply:
Sorry I mixed it up a little bit, I was thinking about the change we did for master before where we moved the V1 schema to runtime/ as well ... so please ignore this and I'm find with the change :)